rails-commenting.rb

```diff
@@ -33,12 +38,14 @@ def create
 
   def edit
     # ---6)
+    # edits the blog post based on the matching id
```

Comment:
This instance variable is locating the blog post with the id that matches the params id